assets/html/themeswap.js

```diff
@@ -25,8 +33,10 @@ function set_theme_from_local_storage() {
     // To distinguish the default (primary) theme, it needs to have the data-theme-primary
     // attribute set.
     var isprimary = (ss.ownerNode.getAttribute("data-theme-primary") !== null);
+    // If the ss is not primary then use it if the user's preference is towards dark colors
+    if(!isprimary) darkTheme = themename;
```

Comment:
While not a big issue currently, it is possible that you'd have multiple light and dark themes, so this could pick the wrong theme. Maybe we should attach another attribute to mark a primary dark theme or something like that?